Source/Cells/Popover/FORMSelectFieldCell.m

```diff
@@ -68,6 +69,15 @@ - (void)updateWithField:(FORMField *)field {
 - (void)updateContentViewController:(UIViewController *)contentViewController
                           withField:(FORMField *)field {
     self.fieldValuesController.field = self.field;
+
+    if (self.field.values.count <= FORMSelectMaxItemCount) {
+        CGSize currentSize = FORMSelectPopoverSize;
+        CGFloat headerViewHeight = self.fieldValuesController.headerView.frame.size.height;
+        CGFloat labelHeight = round(self.fieldValuesController.headerView.labelHeight);
```

Comment:
That would be a great addition, I can make an issue for that and fix it later on.